web/packages/teleterm/src/ui/services/workspacesService/documentsService/documentsUtils.ts

```diff
@@ -62,3 +71,44 @@ export function getResourceUri(
       return undefined;
   }
 }
+
+/**
+ * getDocumentGatewayTargetUriKind is used when the callsite needs to distinguish between different
+ * kinds of targets that DocumentGateway supports when given only its target URI.
+ */
+export function getDocumentGatewayTargetUriKind(
```

Comment:
Don't we have an almost identical function `getGatewayTargetUriKind`?

Reply:
That's right. Gateways themselves support three different kinds (db, kube, app), but `DocumentGateway` supports only db and app. One is used in `ClustersService` when dealing with gateways, the other in `useGateway` when setting the title for `DocumentGateway`.